packages/ui/__tests__/Checkbox.test.tsx

```diff
@@ -0,0 +1,104 @@
+import React from 'react'
+import { mountAndCheckA11Y } from '@hazelcast/test-helpers'
+
+import { Checkbox } from '../src/Checkbox'
+import { Check, Minus } from 'react-feather'
+import { Error, errorId } from '../src'
+import { v4 as uuid } from 'uuid'
+
+jest.mock('uuid')
+
+const uuidMock = uuid as jest.Mock<ReturnType<typeof uuid>>
+
+describe('Checkbox', () => {
```

Comment:
Should we also test interaction?

Reply:
We will, but let me do that in other PR with a radio button, there are some problems I have with enzyme/jest